crates/chain/src/lib.rs

```diff
@@ -100,3 +100,11 @@ pub mod collections {
 
 /// How many confirmations are needed f or a coinbase output to be spent.
 pub const COINBASE_MATURITY: u32 = 100;
+
+impl<A, IA> From<indexed_tx_graph::ChangeSet<A, IA>>
```

Comment:
Not sure if we need this. If we do it shouldn't be here (it seems to be about `indexed_tx_graph`)